test/integration/imagestream_admission_test.go

```diff
@@ -634,7 +616,7 @@ func retryOnQuotaExceeded(t *testing.T, retries int, fn func() error) error {
 			return err
 		}
 		t.Logf("got quota exceeded error, retrying after sleep; number of retries remaining: %d", retries-i-1)
-		time.Sleep(time.Second)
+		time.Sleep(time.Millisecond * 500)
```

Comment:
I find 1 second too long to wait. Moreover, there was no retry before. Now we have two, which I'd like to bump to three.